macos-ignorecase.md

```diff
@@ -0,0 +1,17 @@
+# case-insensitive filesystem on macOS
+
+By default filesystem on macOS is not case sensitive, for new (exlicitly created or copied/moved) files or directories the case of the filename is preserved when listing files, but it's ignored when you want to access specific file e.g. opening/deleting/moving. The situation might be a bit different when you refer to the file in code e.g. javascript import in react-native is case sensitive, because bundler reads exisiting files with correct case and all other operation happens inside the bundler in case sensitive way without filesystem interactions. If you create file `Example.tsx` importing it as `'./example'` won't work in react-native, but it would work in `node` becuse there is no bundler there and all imports use filesystem directly.
```

Comment:
```suggestion
The default macOS filesystem is case-insensitive. This means that for any new file or directory (explicitly created or copied/moved), the casing is preserved when listing files but it's ignored when you want to access the file. The situation might be a bit different when you refer to the file in your code, e.g. imports in React Native are case-sensitive. If you create file `Example.tsx` and import it as `'./example'`, it won't work in React Native. However, it would work in node.
```